src/Model/Gallery.php

```diff
@@ -162,13 +163,36 @@ public function getGalleryHasMedias()
     /**
      * {@inheritdoc}
      */
-    public function addGalleryHasMedias(GalleryHasMediaInterface $galleryHasMedia)
+    public function addGalleryHasMedia(GalleryHasMediaInterface $galleryHasMedia)
     {
         $galleryHasMedia->setGallery($this);
 
         $this->galleryHasMedias[] = $galleryHasMedia;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function removeGalleryHasMedia(GalleryHasMediaInterface $galleryHasMedia)
+    {
+        $this->galleryHasMedias->removeElement($galleryHasMedia);
+    }
+
+    /**
+     * @deprecated use addGalleryHasMedia method instead
+     * NEXT_MAJOR: remove this method with the next major release
+     *
+     * @param GalleryHasMediaInterface $galleryHasMedia
```

Comment:
param is not needed as it is the same as in method

Reply:
I added param here as I'm suggesting to remove this method from GalleryInterface.
Removing a method from interface isn't a breaking change from a practical point of view (it won't break any existing code).